### PR TITLE
Correct grammar; and remove redundant comment

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by Manishearth** [comment]
btw, you don't have to `r?` _every_ PR at me, it's better to just leave the randomly-assigned assignee as-is unless there's a reason to assign me :smile: 

Also, `git push -f` can update branches without needing deletes.

File: src/libcore/slice.rs [hunk 1]
```diff
@@ -622,7 +621,7 @@ impl<T> ops::Index<ops::RangeFrom<usize>> for [T] {
 
 /// Implements slicing with syntax `&self[..]`.
 ///
-/// Returns a slice of the whole slice. This operation can not panic.
+/// Returns a slice of the whole slice. This operation cannot panic.
```
Comment (Manishearth) [inline src/libcore/slice.rs]: This isn't a grammar fix, both are correct

Comment (srinivasreddy) [inline src/libcore/slice.rs]: Thanks.

**Pull request merged**